shell/assets/styles/themes/_light.scss

```diff
@@ -625,4 +625,13 @@ BODY, .theme-light {
   --click-badge-info-bg        : #{$info};
   --click-badge-info-border    : #2673A6;
   --bubble-bg                : #{$lighter};
+
+
+  // These are meant to be the colors used within rancher-components. They will eventually be removed from shell once we have theming working in the components
+  --rc-info: #1F67DB;
+  --rc-success: #007032;
+  --rc-warning: #FFE47A;
+  --rc-error: #B13333;
+  --rc-unknown: #EDEFF3;
+  --rc-none: #DCDEE4;
```

Comment:
These are the colors that are used in the DS but I've been told that they will change.

Reply:
Agreed. Just not sure when it's going to be ready
<img width="948" height="630" alt="image" src="https://github.com/user-attachments/assets/2f0952bd-d60c-4a81-b6b0-14480fd76ab3" />

Reply:
@rak-phillip given the answer above, and in an effort to keep things moving. 

Would you like to maybe create a `colors.scss` which we can bind the temporary hex values to a name and then replace those vars everywhere we use them once it gets finalized? If you have another idea I'd be open to that as well.

Reply:
> Would you like to maybe create a colors.scss which we can bind the temporary hex values to a name and then replace those vars everywhere we use them once it gets finalized?

Yeah, I think that makes sense. With my original comment being a nit, I don't think it should block moving forward with this PR. I think we have a path forward for adding some structure around this area.